server/src/controllers/Events/resolver.ts

```diff
@@ -365,27 +330,18 @@ export class EventResolver {
     });
   }
 
-  // TODO: Check we need all the returned data
   @Authorized(Permission.EventEdit)
-  @Query(() => EventWithRelations, { nullable: true })
+  @Query(() => EventWithRelationsWithEventUserRelations, { nullable: true })
   async dashboardEvent(
     @Arg('id', () => Int) id: number,
-  ): Promise<EventWithRelations | null> {
+  ): Promise<EventWithRelationsWithEventUserRelations | null> {
```

Comment:
I see what you mean about the name, but it's okay. At least, I can't come up with good names for these, either.